stdlib/runtime/numberUtils.gr

```diff
@@ -1148,46 +1244,60 @@ let genExponent = (buffer, k) => {
   if (sign) k = 0n - k
   let decimals = decimalCount32(k) + 1n
   utoa32_dec_core(buffer, k, decimals)
-  WasmI32.store8(buffer, (if (sign) _CHAR_CODE_MINUS else _CHAR_CODE_PLUS), 0n)
+  WasmI32.store8(buffer, if (sign) _CHAR_CODE_MINUS else _CHAR_CODE_PLUS, 0n)
   decimals
 }
 
 let grisu2 = (value, buffer, sign) => {
   // frexp routine
-  let uv  = WasmI64.reinterpretF64(value)
-  let mut exp = WasmI32.wrapI64(WasmI64.shrU(WasmI64.and(uv, 0x7FF0000000000000N), 52N))
+  let uv = WasmI64.reinterpretF64(value)
+  let mut exp = WasmI32.wrapI64(
+    WasmI64.shrU(WasmI64.and(uv, 0x7FF0000000000000N), 52N),
+  )
   let sid = WasmI64.and(uv, 0x000FFFFFFFFFFFFFN)
-  let mut frc = WasmI64.add(WasmI64.shl((if (WasmI32.eqz(exp)) 0N else 1N), 52N), sid)
+  let mut frc = WasmI64.add(
+    WasmI64.shl(if (WasmI32.eqz(exp)) 0N else 1N, 52N),
+    sid,
+  )
   exp = (if (WasmI32.eqz(exp)) 1n else exp) - (0x3FFn + 52n)
 
   // normalize boundaries
   let mut frc_norm = WasmI64.add(WasmI64.shl(frc, 1N), 1N)
   let mut exp_norm = exp - 1n
   let off_norm = WasmI64.clz(frc_norm)
   frc_norm = WasmI64.shl(frc_norm, off_norm)
-  exp_norm  -= WasmI32.wrapI64(off_norm)
+  exp_norm -= WasmI32.wrapI64(off_norm)
 
   let m_norm = 1n + (if (WasmI64.eq(frc, 0x0010000000000000N)) 1n else 0n)
 
-  let _frc_plus  = frc_norm
-  let _frc_minus = WasmI64.shl(WasmI64.sub(WasmI64.shl(frc, WasmI64.extendI32U(m_norm)), 1N), WasmI64.extendI32U(exp - m_norm - exp_norm))
+  let _frc_plus = frc_norm
+  let _frc_minus = WasmI64.shl(
+    WasmI64.sub(WasmI64.shl(frc, WasmI64.extendI32U(m_norm)), 1N),
+    WasmI64.extendI32U(exp - m_norm - exp_norm),
+  )
   let _exp = exp_norm
 
   // get cached power
-  let c = WasmF64.reinterpretI64(0x3FD34413509F79FEN) // 1 / lg(10) = 0.30102999566398114
-  let dk = WasmF64.add(WasmF64.mul(WasmF64.convertI32S(-61n - _exp), c), 347.0W) // dk must be positive, so can do ceiling in positive
+  let c = WasmF64.reinterpretI64(
+    0x3FD34413509F79FEN,
+  ) // 1 / lg(10) = 0.30102999566398114
+  let dk = WasmF64.add(
+    WasmF64.mul(WasmF64.convertI32S(-61n - _exp), c),
+    347.0W,
+  ) // dk must be positive, so can do ceiling in positive
   let mut k = WasmI32.truncF64S(dk)
-  k += if (WasmF64.ne(WasmF64.convertI32S(k), dk)) 1n else 0n // conversion with ceil
+  k += if (WasmF64.ne(WasmF64.convertI32S(k), dk)) 1n
+  else 0n // conversion with ceil
```

Comment:
Need to indent this too?

Reply:
We do, that's a piece of syntactic sugar I need to stop to handle it.

Reply:
Fixed.

Reply:
This doesn't looked fixed. Since it is doing an assignment, the `else` should be indented.